docs/admin/50-resources/10-whats-new.md

```diff
@@ -4,6 +4,20 @@ id: whats-new
 title: "What's New"
 ---
 
+# 2025 March 26
+
+**Release notes OpenCloud 2.0.0 (Production release)**
+
+- **🌲 File Native Backups with PosixFS**: OpenCloud introduces PosixFS to allow for easy file native backups. As all files and metadata information is stored directly on the storage (no database needed), you can create standard file-level snapshots or copies. <br />
+Previously, files and folders were stored in a hash-based structure, making the layout unreadable for humans.With PosixFS, the full file tree is directly visible on the storage, providing transparency and confidence that all data is being backed up correctly. With this, we would like that admins benefit from easier maintenance, fewer moving parts, and improved fault tolerance in backup and disaster recovery scenarios. <br />
```

Comment:
```suggestion
Previously, files and folders were stored in a hash-based structure, making the layout unreadable for humans. With PosixFS, the full file tree is directly visible on the storage, providing transparency and confidence that all data is being backed up correctly. Admins benefit from easier maintenance, fewer moving parts, and improved fault tolerance in backup and disaster recovery scenarios. <br />
```

IMO the extra words "we would like that" etc. are not really needed, and don't quite read easily.